hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/KafkaOffsetPostProcessor.java

```diff
@@ -36,15 +36,8 @@
 public class KafkaOffsetPostProcessor extends SchemaPostProcessor {
 
   public static class Config {
-    public static final ConfigProperty<String> KAFKA_APPEND_OFFSETS = ConfigProperty
```

Comment:
similar here, annotate with `@deprecated` and do `KAFKA_APPEND_OFFSETS = HoodieDeltaStreamerConfig.KAFKA_APPEND_OFFSETS;`

Reply:
Fixed.